xem

```diff
@@ -20,6 +20,7 @@ xem is a Scala program wrapped in a shell script. It takes the source files dire
         --mask-elements             [a comma-separated list of XML element tags, whose content and sub elements should be masked with CDATA]
         --target-files-directory    [ABSOLUTE PATH TO TARGET FILES DIRECTORY]
         --content-only              utilise only the content of the sub elements (i.e. without further XML elements tags inside)
+        --exclude-empty-elements    exclude empty XML element tags, which should be masked with CDATA
```

Comment:
I would recommend to make this handling mandatory (instead of optional via a command argument), since it is, afaik, necessary to (hopefully) produce valid XML.